packages-next/types/src/config/index.ts

```diff
@@ -177,6 +178,13 @@ export type ImagesConfig = {
   };
 };
 
+// config.testing
+
+export type TestingConfig = {
+  isolateRuns?: Boolean;
```

Comment:
This name feels misleading since it's really "isolate unique prisma schemas" not isolating every run

Reply:
Also, 
```suggestion
  isolateRuns?: boolean;
```

Reply:
Yep, fair call, open to suggestions 🤔

Reply:
`isolateSchemas`? `isolateConfigs`? I think I like `isolateConfigs`.